src/dll/Hooks/ValidateScripts.cpp

```diff
@@ -37,10 +36,11 @@ bool _ScriptValidator_Validate(uint64_t self, uint64_t a1, RED4ext::ScriptReport
         }
     }
 
-    auto message = WritePopupMessage(errors);
+    const auto& incompatiblePlugins = App::Get()->GetPluginSystem()->GetIncompatiblePlugins();
+    const auto message = WritePopupMessage(validationErrors, incompatiblePlugins);
     if (!message.empty())
     {
-        MessageBoxA(0, message.c_str(), "Script Validation Error", MB_OK | MB_ICONERROR);
+        SHOW_MESSAGE_BOX_AND_EXIT_FILE_LINE("{}", message);
```

Comment:
```suggestion
        SHOW_MESSAGE_BOX_AND_EXIT_FILE_LINE(message);
```

Reply:
this doesn't compile, I think the first argument must be a format string

Reply:
not really format string but it must be string literal, the way bopps uses it, looking at the source code.

but what you did is an error, as bopps doesnt use explicit format args placement so it would corrupt things...

i guess this function is unusable for you and what you had before is a better choice... or this macro has to be modified in some way as there is no way to pass this properly into it without changing the macro...

Reply:
diregard, invalid comment... deleted it...

yeah, it must be how you call it for it to function, this is good...

it expects tring literal there

Reply:
bopps prolly didnt realise so yeah, you had it good

Reply:
should be L"{}" (with prefix) but other than that, i dont see an issue with that call

Reply:
> bopps prolly didnt realise so yeah, you had it good

I didn't realize it :(

But yes, `L"{}"` should be there.